the_debuginator.h

```diff
@@ -1304,11 +1317,38 @@ bool debuginator__distance_to_hot_item(DebuginatorItem* item, DebuginatorItem* h
 
 			child = debuginator__next_visible_sibling(child);
 		}
+	} else if (item->leaf.is_expanded && !item->is_filtered) {
+		*distance += item_height * item->leaf.num_values;
+		*distance += item->leaf.description_line_count * item_height;
 	}
-
 	return false;
 }
 
+int debuginator_total_height(TheDebuginator* debuginator) {
```

Comment:
This is used by the mouse support, and I appear to have messed up including it here. If I recall correctly the reason to use this was that total height was not updated properly when filtered, so I calculated it every time. Now that it _is_ calculated on filtering, using total height on root might work. Would need to test it though, not sure if expanding things properly propagates total height up the tree.